package.json

```
@@ -22,7 +22,8 @@
     "start:stub": "docker-compose -f docker-compose.stub.yml up",
     "license-check": "license-check-and-add check -f license-config.json",
     "license-add": "license-check-and-add add -f license-config.json",
-    "license-remove": "license-check-and-add remove -f license-config.json"
+    "license-remove": "license-check-and-add remove -f license-config.json",
+    "rerun-price-calculations": "ts-node src/scripts/rerunPriceCalculations.ts"
```

```suggestion
    "fix:price-calculations": "ts-node src/scripts/rerunPriceCalculations.ts"
```
The idea is if we have more than one action that fixes something, we will again use the prefix `fix:`